brands/amenity/bank.json

```diff
@@ -5170,6 +5190,17 @@
       "name": "Privatbanka"
     }
   },
+  "amenity/bank|Promerica": {
```

Comment:
For this one I notice:
* It operates in countries other than just Honduras (I found https://www.promerica.fi.cr/, not sure how these brands are related)
* identifier and `brand` should probably consistently be either "Promerical" or "Banco Promerica" (with matchName for whatever the less preferred alternative is)
* need to replace underscores with spaces in the `brand:wikipedia` tag